documentation/docs/opal-plus/features.mdx

```diff
@@ -0,0 +1,76 @@
+---
+sidebar_position: 3
+title: Advanced Features
+---
+
+OPAL+ has a number of advanced features extending the capabilities of the open-source OPAL.
+These features are available to OPAL+ users only.
+
+To apply for Permit OPAL+, [fill in the form available here](https://hello.permit.io/opal-plus)
+
+
+## Support and SLA
+
+OPAL+ provides dedicated support and a custom SLA to help you get the most out of your OPAL+ deployment.
+Reach out to us [on Slack](https://bit.ly/permit-slack) for more information.
+
+## Licensing Capabilities
+
+OPAL+ provides additional licensing capabilities to help you manage your OPAL+ deployment.
+Reach out to us [on Slack](https://bit.ly/permit-slack) for more information.
+
+## Logging and Monitoring
+
+OPAL+ provides advanced logging and monitoring capabilities to help you track and debug your OPAL+ deployment.
+
+#### Connect to logging system
+
+On production, we advise you to connect OPAL+ to your logging system to collect and store the logs.
+Configure the [OPAL_LOG_SERIALIZE](/getting-started/configuration) environment variable to `true` to serialize logs in JSON format.
+
+#### Monitor OPAL Servers and Clients
+
+OPAL+ provides monitoring endpoints to help you track the health of your OPAL+ servers and clients.
+Configure the [OPAL_STATISTICS_ENABLED=true](/getting-started/configuration) environment variable to enable the statistics APIs.
+
+You can then monitor the state of your OPAL+ cluster by calling the `/stats` API route on the server.
+```bash
+curl http://opal-server:8181/stats -H "Authorization: Bearer <token>"
+# { "uptime": "2024-07-14T14:55:02.710Z", "version": "0.7.8", "client_count": 1, "server_count": 1 }
+```
+
+You can also get detailed information about the OPAL+ clients and servers by calling the `/statistics` API route on the server.
```

Comment:
@orweis should we ask them in those docs to share the output with us for licensing?